xml/art_amd-sev.xml

```diff
@@ -70,11 +70,6 @@
   halted. SEV-ES builds upon SEV to provide an even smaller attack surface for
   virtual machines running in a multi-tennant environment.
  </para>
- <para>
```

Comment:
Maybe we can add a table with information of which SLE versions have which level of support for SEV/SEV-ES?

Reply:
Does that information exist somewhere?

Reply:
According to @aginies , SEV is supported on 15 SP4 only